lib/features/family/features/reflect/presentation/pages/gratitude_selection_screen.dart

```diff
@@ -22,10 +24,20 @@ class GratitudeSelectionScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return FunScaffold(
-      appBar: FunTopAppBar.primary99(title: 'Gratitude'),
+      appBar: FunTopAppBar.primary99(
+        title: 'Gratitude',
+        actions: [
+          IconButton(
```

Comment:
I would extract this in a widget so it can be re-used across the screens (for example in the GratefulScreen)

Reply:
I extracted to button and changed it on all pages were this button is used :)